packages/orn-locator/src/resolvers/books.ts

```diff
@@ -117,6 +118,7 @@ const commonBook = memoize(async(id: string, version?: string, archive?: string)
       type: 'book' as const,
       state: oswebData.book_state as string,
       title: oswebData.title as string,
+      subject: subject as string,
```

Comment:
we discussed passing through all the subjects, but remapping the format a bit

Reply:
And I'm also going to return the categories to give us flexibility

Reply:
@TomWoodward Maybe you can help me design our ideal output here... categories do give us the subject name, so I could return just categories. Here's an example with both fields, using one of the rare books that have multiple:

```
"subjects": [
  {
    "id": 386,
    "name": "Social Sciences"
  },
  {
    "id": 388,
    "name": "Science"
  }
],
"categories": [
  {
    "id": 205,
    "name": "Biology",
    "subject_name": "Science"
  },
  {
    "id": 207,
    "name": "Psychology",
    "subject_name": "Social Sciences"
  }
],
```

Reply:
Though the order inversion there is interesting... if I need Social Sciences for a filter, I'm really not sure what is reliable. Maybe I should use the subject after all.

Reply:
@TomWoodward Boop

Reply:
i think the output you pasted looks good, better to have both i think